cmd/test.go

```diff
@@ -199,8 +238,22 @@ func init() {
 	TestCmd.Flags().StringVar(&metadata, "metadata", "", "path to image metadata file")
 	TestCmd.Flags().BoolVar(&pull, "pull", false, "force a pull of the image before running tests")
 	TestCmd.Flags().BoolVar(&save, "save", false, "preserve created containers after test run")
-	TestCmd.Flags().BoolVar(&force, "force", false, "force run of host driver (without command line input)")
 	TestCmd.Flags().BoolVar(&quiet, "quiet", false, "flag to suppress output")
-	TestCmd.Flags().BoolVarP(&verbose, "verbose", "v", false, "verbose testing output")
+	TestCmd.Flags().BoolVar(&force, "force", false, "force run of host driver (without user prompt)")
+
 	TestCmd.Flags().StringArrayVar(&configFiles, "config", []string{}, "test config files")
 }
+
+func isQuiet() bool {
+	return quiet
+}
+func initTemplateFuncMap() map[string]interface{} {
```

Comment:
what's this for?

Reply:
This is to make the Functions available inside the template.
https://golang.org/pkg/text/template/#FuncMap